core/tests/test_apps.cardano.address.py

```diff
@@ -26,9 +26,9 @@ def test_hardened_address_derivation_scheme(self):
         mnemonic = "all all all all all all all all all all all all"
         passphrase = ""
         node = bip32.from_mnemonic_cardano(mnemonic, passphrase)
-        node.derive_cardano(0x80000000 | 44)
-        node.derive_cardano(0x80000000 | 1815)
-        keychain = Keychain([0x80000000 | 44, 0x80000000 | 1815], node)
+        node.derive_cardano(44 | HARDENED)
+        node.derive_cardano(1815 | HARDENED)
+        keychain = Keychain([44 | HARDENED, 1815 | HARDENED], node)
```

Comment:
Why don't we use BYRON_PURPOSE etc.?

Reply:
Because I think in the tests it is more readable and preferable not to have the constant. And it is very clear what it means.